.docker/aiida-core-base/s6-assets/config-profile-setup.yml

```diff
@@ -0,0 +1,3 @@
+postgres_hostname: database
```

Comment:
This is correct change, in docker compose the hostname is `database`. 
However, with docker compose, from aiida container I run `pg_isready -h database` which shows the database is ready. Running quicksetup directly still failed.

Reply:
The problem is that that fixes PSQL but there is still a problem for RMQ whose hostname is `messaging` but `verdi presto` checks it under `localhost`. There is no option to configure it though, on purpose. Is there a way in Docker to "forward" the hostname to localhost?

Reply:
But why the previous images works? It uses localhost instead of `messaging`.

Reply:
>  Is there a way in Docker to "forward" the hostname to localhost?

Maybe this will work? https://docs.docker.com/compose/compose-file/05-services/#hostname 
I think the more standard way of using docker compose is through `network` setting with alias, see https://stackoverflow.com/a/47264089.

Reply:
> But why the previous images works? It uses localhost instead of `messaging`.

Not sure. The previous commit `aa3337a` the aiida-core-with-services worked, but in `337d520` it failed. It failed because the daemon could not be reached though, the profile was setup fine and could connect to the services:
```
 ✔ config:      /home/aiida/.aiida
 ✔ profile:     presto
 ✔ storage:     Storage for 'presto' [open] @ ***localhost:5432/aiida-presto / DiskObjectStoreRepository: 2bb2bdb3bf804a3a9e50283fcd34f8d8 | /home/aiida/.aiida/repository/presto/container
 ✔ broker:      RabbitMQ v3.10.18 @ ***localhost:5672?heartbeat=600
 ✘ daemon:      Connection to the daemon timed out.
```
Might have just been a fluke